cloud/jenkins/psmdb_operator_aws_openshift-4.groovy

```diff
@@ -501,17 +514,10 @@ pipeline {
                         clusterRunner('cluster4')
                     }
                 }
```

Comment:
Why do we remove cluster5?

Reply:
Because we did this for `psmdb_operator_aws_openshift_latest.groovy` some time in the past with the assumption that there were too many clusters running in parallel causing problems. So, I want to have `xxx-latest` and `xxx-version` to be exactly the same.